docs/components/page_links.tsx

```diff
@@ -0,0 +1,25 @@
+import * as styles from "df/docs/components/page_links.css";
+import React from "react";
+
+export interface IHeaderLink {
+  id: string;
+  text: string;
+}
+
+interface IProps {
+  links: IHeaderLink[];
+}
+
+export class PageLinks extends React.Component<IProps> {
```

Comment:
In general for small bits like this I would use a functional component instead of a class, less code and easier to think about.

Reply:
My general position on this has been to always do the same thing everywhere. However, I will try it, for you.